### DISCUSSION TITLE
Essentials cleanup

### DISCUSSION THREAD

**Discussion by Teense** [EssentialsX/Essentials#4059] [Q&A] [answered]
Hello, I recently used the /essentials cleanup command and have been showing errors in the console ever since. I found out that these are errors, because the command only deleted userdata, but did not delete it in usermap.cvs, is it possible to delete it later?

Thanks,
Teense_

**Comment by JRoy** [comment, marked as answer]
Yes

**Reply by Teense** [reply]
How?
